src/tsickle.ts

```diff
@@ -677,12 +677,8 @@ class Annotator extends ClosureRewriter {
     if ((classDecl.flags & ts.NodeFlags.Abstract) !== 0) {
       jsDoc.push({tagName: 'abstract'});
     }
-    if (jsDoc.length === 0) {
-      this.writeRange(classDecl.getFullStart(), classDecl.getStart());
```

Comment:
This was kind of nice to include any comments before the JSDoc. But I guess we don't really care?